src/adb/devices.js

```diff
@@ -1,8 +1,7 @@
 let debug = console.log.bind(console, '[adb/devices]');
-let exec = require('mz/child_process').exec;
 
 module.exports = async function devices() {
-  let output = await exec('adb devices');
+  let output = await this.exec('devices');
```

Comment:
One thing is that you wouldn't be able to use `adb/devices` without calling it through `adb/index` with thing. That might be okay but something to consider